pkgs/servers/pufferpanel/default.nix

```diff
@@ -46,50 +61,52 @@ buildGoModule rec {
     "-X=github.com/pufferpanel/pufferpanel/v2.Version=${version}-nixpkgs"
   ];
 
-  src = fetchFromGitHub {
-    owner = "pufferpanel";
-    repo = pname;
-    rev = "v${version}";
-    hash = "sha256-0Vyi47Rkpe3oODHfsl/7tCerENpiEa3EWBHhfTO/uu4=";
-  };
+  # Web UI is platform-independent but NodeJS fails to cross-compile at least
+  # on `aarch64-linux`-to-`x86_64-linux` builds.
+  frontend = pkgsBuildBuild.buildNpmPackage {
```

Comment:
I’m not really sure if this should be `buildPackages` or `pkgsBuildBuild`, and whether using one over another would make a difference in this case.

```
$ rg buildPackages | wc -l
1210
$ rg pkgsBuildBuild | wc -l
115
```

Apparently `pkgsBuildBuild` is not used that much?

@SuperSandro2000 could you help me with that?

Reply:
Those variables are for getting build or runtime versions of packages where splicing does not work aka in strings. We can take buildNpmPackage from inputs.

Reply:
```suggestion
  frontend = buildNpmPackage {
```